Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
```

Comment:
### [commander](https://github.com/commander-rb/commander)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 4.4.3 → 4.4.7

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **28** commits. See the full changes on [the compare page](https://github.com/commander-rb/commander/compare/v4.4.3...v4.4.7).

These are the first 10 commits:
- (c5e5b12) [README: Use SVG for build badges](https://github.com/commander-rb/commander/commit/c5e5b1276c69f2a02551ccb39d3283644691d291)
- (fefd848) [Merge pull request #49 from olleolleolle/patch-1](https://github.com/commander-rb/commander/commit/fefd8480ce0e3fe716ea14ed848bd8c65f58708e)
- (bade360) [rubocop: tunes rubocop categories as they should](https://github.com/commander-rb/commander/commit/bade360d2ab7aea81716b19e36f5c959aa91b45a)
- (fd8376b) [rubocop: tells minimum supported ruby version](https://github.com/commander-rb/commander/commit/fd8376b0d4de20be9d94215989f39f9bec16c5dc)
- (b620792) [rubocop: disable cops not compatible with ruby < 2.0](https://github.com/commander-rb/commander/commit/b620792db560841ab12e036f9d44a1551b22afb2)
- (90174cc) [rubocop: please me](https://github.com/commander-rb/commander/commit/90174cc8dca099e87ee300428d56b6b789bc1dc2)
- (395bf92) [Merge pull request #55 from lacostej/rubocop_fixes](https://github.com/commander-rb/commander/commit/395bf920713dbb2eac5a98674b7c35cf140f9fea)
- (382dcc8) [README: describe summary field](https://github.com/commander-rb/commander/commit/382dcc80a0e17091bb02f1450744a582d970d12c)
- (438e60f) [Merge pull request #56 from lacostej/summary](https://github.com/commander-rb/commander/commit/438e60ff9ef163729872031325cf850ae88ec821)
- (bcf2425) [Update Travis badge link](https://github.com/commander-rb/commander/commit/bcf242574b81a07b4117b0af8ecf22dcf40e0d9c)


</details>